packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -30,6 +30,20 @@ export interface BatchesTableProps {
   rows: BatchRow[];
 }
 
+export const sortByExpiry = (a: BatchRow, b: BatchRow) => {
```

Comment:
I had meant to do the sorting which you mentioned ages ago. thanks for getting to it